packages/ketcher-polymer-editor-react/src/components/shared/ui/Tabs/Tabs.tsx

```diff
@@ -13,13 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  ***************************************************************************/
-import * as React from 'react'
-import Tabs from '@mui/material/Tabs'
-import Tab from '@mui/material/Tab'
-import { useState } from 'react'
-import clsx from 'clsx'
-import classes from './Tabs.module.less'
+
+import Tabs, { TabsProps } from '@mui/material/Tabs'
+import Tab, { TabProps } from '@mui/material/Tab'
+import { ReactElement, useState } from 'react'
 import Box from '@mui/material/Box'
+import { styled } from '@mui/material'
```

Comment:
Though they're quire similar, we need `styled` from `@emotion/styled`, not mui :)